user-creation/src/main/java/no/unit/nva/useraccessservice/usercreation/person/NationalIdentityNumber.java

```diff
@@ -8,6 +8,10 @@ public NationalIdentityNumber(final String nin) {
         this.nin = nin;
     }
 
+    public static NationalIdentityNumber fromString(final String nin) {
+        return new NationalIdentityNumber(nin);
+    }
```

Comment:
How is this different from the construtor? Both are public. Maybe make the constructor private, or use the constructor directly instead?